src/TimeSpans.jl

```diff
@@ -371,6 +371,7 @@ Return a vector of `TimeSpan`s representing the gaps between the spans in the
 iterable `spans` that are contained within `parent_span`.
 """
 function invert_spans(spans, parent_span)
+    isempty(spans) && return TimeSpan[parent_span]
```

Comment:
Maybe this is better? So we hit the same `start` and `stop` (via the constructor) as in the other branch, rather than a `convert`:
```suggestion
    isempty(spans) && return[TimeSpan(parent_span)]
```

Reply:
I actually had that initially and changed it but I have no idea why